README.md

```diff
@@ -299,7 +299,7 @@ contribute.
     [our Carbon release post on GitHub](https://github.com/carbon-language/carbon-lang/discussions/1020)
     and [star carbon-lang](https://github.com/carbon-language/carbon-lang).
 -   To join the design discussion, join our
-    [our GitHub forum](https://github.com/carbon-language/carbon-lang/discussions).
+    [GitHub forum](https://github.com/carbon-language/carbon-lang/discussions).
```

Comment:
FYI, I believe this was separately fixed in another PR if you merge from trunk.

Reply:
@jonmeow I forked from the trunk today.

Can you help me solve 'pre-commit' problem in windows.

Reply:
@jonmeow I have updated and formatted. Used your Suggested Commit. Please have a look

Reply:
@jonmeow This feel kind of odd. When some one made pull same pull request after me and get merged.
[gh pr checkout 1631](https://github.com/carbon-language/carbon-lang/pull/1631)

Reply:
Sorry for any confusion, just a bunch of different folks helping spot and fix the same typos and bugs at the same time. =D We still totally appreciate the contributions and help.